src/clear/model/modules/has_columns.cr

```diff
@@ -146,7 +146,10 @@ module Clear::Model::HasColumns
   #   when instantiating or updating a new model from json through `.from_json` methods from
   #   the `Clear::Model::JSONDeserialize` module.
   #
-  macro column(name, primary = false, converter = nil, column_name = nil, presence = true, mass_assign = true)
+  # * `write_only : Bool (default = true)`: same as `ignore_serialize`: turn on/ off serialization
+  #   of a field when doing `.to_json` on the model
+  #
```

Comment:
This doesn't make sense to me. `write_only` implies you cannot read the value. No point overloading/changing the semantics of `ignore_serialize` if that's what it implies.

Reply:
This is mainly for OpenAPI and partially for anything else. `write-only` is OpenAPI's terminology.

Reply:
In that case, better to do it where there's an interface between the two libraries, or via tags that don't impact serialisation.
These are two separate concerns.

A general user of `clear` probably doesn't want to buy into OpenAPI tags that don't particularly make sense in the context of the library for their use case